solc/CommandLineInterface.cpp

```diff
@@ -691,6 +723,12 @@ Allowed options)",
 		return false;
 	}
 
+	if (!setupTermColors(m_args.count(g_argColor), m_args.count(g_argNoColor)))
+	{
+		serr() << "Option " << g_argColor << " and " << g_argNoColor << " are mutualy exclusive." << endl;
```

Comment:
Isn't there a boost program options feature for that?

Reply:
I could only find one for flags in general.

But what I need is some kind of an **optional** flag (like a tribool), so I can reflect (yes, no, auto-detect).